hrabsence/tests/phpunit/api/v3/ActivityGetAbsencesTest.php

```diff
@@ -239,4 +239,49 @@ public function testOptionAbsenceRange() {
       }
     }
   }
+
+  public function testStatusIdParameter() {
```

Comment:
Could you split these test into 3 different tests (with meaningful method names), one for each assertion blocks? It improves readability and structure of the test suite